doc/user_guide/Customization.ipynb

```diff
@@ -210,7 +210,15 @@
     "        the number of individual data points present in the current zoom range\n",
     "        is above this threshold. The raw plot is displayed otherwise.\n",
     "    x_sampling/y_sampling (default=None):\n",
-    "        Specifies the smallest allowed sampling interval along the x/y axis."
+    "        Specifies the smallest allowed sampling interval along the x/y axis.\n",
+    "    pixel_ratio (default=None):\n",
+    "        Pixel ratio applied to the height and width. Useful for higher\n",
```

Comment:
Pointed out by @maximlt, we should also mention this is applicable for datashader operations here and in the docstring.

Bonus if you also mention it for x_sampling/y_sampling!

Reply:
Done! Let me know what you think of the addition.